src/Microsoft.Azure.ServiceBus/Message.cs

```diff
@@ -86,12 +83,10 @@ public string PartitionKey
 
         /// <summary>Gets or sets a partition key value when a transaction is to be used to send messages via a transfer queue.</summary>
```

Comment:
> when a transaction is to be used to send [](start = 55, length = 41)

use similar language as above for PartitionKey.like "for sending a transactional message via a transfer queue"